app/src/main/java/io/gnosis/safe/ui/settings/app/SettingsHandler.kt

```diff
@@ -45,6 +48,18 @@ class SettingsHandler @Inject constructor(
         }
     }
 
+    var trackingAllowed: Boolean
+        get() = preferencesManager.prefs.getBoolean(KEY_ALLOW_TRACKING, true)
+        set(value) {
+            preferencesManager.prefs.edit {
+                putBoolean(KEY_ALLOW_TRACKING, value)
+            }
+        }
+
+    fun allowTracking(context: Context, allow: Boolean) {
+        FirebaseAnalytics.getInstance(context).setAnalyticsCollectionEnabled(allow)
```

Comment:
Do we track crashes regardless of this setting?

Reply:
Will check. With the new crash button :-)

Reply:
Deliberate crashes are still reported. Even if tracking is off. I think it safe to assume that any crash is still reported.